### DISCUSSION TITLE
Feature Request: Minimizable Dialog Component/Option

### DISCUSSION THREAD

**Discussion by sja-cslab** [primefaces/community#1788] [PrimeReact] [answered]
Hey everyone,

I've been thinking about a neat addition to PrimeReact that could make a big difference in user experience: a minimizable option for Dialog components.

The idea is to give users the ability to minimize Dialogs, which would be super handy in scenarios where users need to focus on other tasks or refer to information elsewhere on the page without closing the Dialog entirely.

I envision a simple button or icon in the Dialog's title bar that toggles between minimizing and restoring the Dialog's visibility.

When minimized, the Dialog could display as a smaller version, perhaps with just the title bar visible, showing an indicator that there's content inside. This would allow users to easily see and access the Dialog when needed, without it taking up too much space or being distracting.

**Comment by sja-cslab** [comment]
@melloware I'm always interested in your opinion.

**Reply by melloware** [reply]
PrimeFaces JSF has a minimize you can see it in action here if you click the "Minimize and Maximize" example: https://www.primefaces.org/showcase/ui/overlay/dialog.xhtml

**Reply by sja-cslab** [reply]
So what about PrimeReact? Any plans?

**Reply by melloware** [reply]
For that I do not know if PrimeTek has any plans for minimize for the dialogs.  Typically they try to align their 3 JS libs for features like this.  You should check PrimeNG or PrimeVue to see if someone has requested a similar feature?

**Reply by sja-cslab** [reply]
Did that already and just found someone [who tries to do it by himself](https://github.com/primefaces/primeng/issues/15384).
So how does this feature requests issue exactly work? Noticed you just openend another feature request within issues on PrimeReact!?

**Reply by melloware** [reply, marked as answer]
yeah i think you can just create a GitHub issue as a New Feature.

**Reply by sja-cslab** [reply]
[Done](https://github.com/primefaces/primereact/issues/6491)
